docs/running-on-kubernetes.md

```diff
@@ -602,4 +602,52 @@ specific to Spark on Kubernetes.
    <code>spark.kubernetes.executor.secrets.spark-secret=/etc/secrets</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.driver.volumes.[VolumeType].[VolumeName].mount.path</code></td>
```

Comment:
It's worth mentioning that `VolumeType` should be a type supported by Kubernetes and use the same type names as those used by Kubernetes.

Reply:
This is mentioned higher up in documentation. I'm not sure if it should be repeated:
```
Add the <a href="https://kubernetes.io/docs/concepts/storage/volumes/">Kubernetes Volume</a> named <code>VolumeName</code> of the <code>VolumeType</code> type to the driver pod on the path specified in the value.
```